src/main/java/views/LeagueSimulatorPane.java

```diff
@@ -0,0 +1,296 @@
+                "        -fx-spacing: 15;");
+
+        matchesContainer.setAlignment(Pos.CENTER);
+
+        title.setStyle("-fx-font-size: 20;");
+        sizeLabel.setStyle("-fx-font-size: 16;");
+
+        String bigButton = "    -fx-font-size: 15;\n" +
+                "    -fx-font-family: sans-serif;\n" +
+                "    -fx-pref-width: 200;\n" +
+                "    -fx-pref-height: 40;\n" +
+                "    -fx-background-radius: 16px;";
+
+        returnButton.setStyle(bigButton);
+        playButton.setStyle(bigButton);
+        refreshButton.setStyle(bigButton);
+    }
+
+    void createLeagueSchedule() throws Exception{
+
+        Queue<Team> schedule = new LinkedList<>();
+        int leagueSize = Integer.parseInt(size.getSelectionModel().getSelectedItem());
+        ArrayList<String> teamNames = new ArrayList<>();
+
+        if ( leagueSize == 8 ){
+            teamNames.addAll(Arrays.asList(comboBox1.getSelectionModel().getSelectedItem(),
+                    comboBox2.getSelectionModel().getSelectedItem(),
+                    comboBox3.getSelectionModel().getSelectedItem(),
+                    comboBox4.getSelectionModel().getSelectedItem(),
+                    comboBox5.getSelectionModel().getSelectedItem(),
+                    comboBox6.getSelectionModel().getSelectedItem(),
+                    comboBox7.getSelectionModel().getSelectedItem(),
+                    comboBox8.getSelectionModel().getSelectedItem(),
+                    comboBox9.getSelectionModel().getSelectedItem(),
+                    comboBox10.getSelectionModel().getSelectedItem(),
+                    comboBox11.getSelectionModel().getSelectedItem(),
+                    comboBox12.getSelectionModel().getSelectedItem(),
+                    comboBox13.getSelectionModel().getSelectedItem(),
+                    comboBox14.getSelectionModel().getSelectedItem(),
+                    comboBox15.getSelectionModel().getSelectedItem(),
+                    comboBox16.getSelectionModel().getSelectedItem()));
+        } else if (leagueSize == 4){
+            teamNames.addAll(Arrays.asList(comboBox1.getSelectionModel().getSelectedItem(),
+                    comboBox2.getSelectionModel().getSelectedItem(),
+                    comboBox3.getSelectionModel().getSelectedItem(),
+                    comboBox4.getSelectionModel().getSelectedItem(),
+                    comboBox5.getSelectionModel().getSelectedItem(),
+                    comboBox6.getSelectionModel().getSelectedItem(),
+                    comboBox7.getSelectionModel().getSelectedItem(),
+                    comboBox8.getSelectionModel().getSelectedItem()));
+
+        } else {
+            teamNames.addAll(Arrays.asList(comboBox1.getSelectionModel().getSelectedItem(),
+                    comboBox2.getSelectionModel().getSelectedItem(),
+                    comboBox3.getSelectionModel().getSelectedItem(),
+                    comboBox4.getSelectionModel().getSelectedItem()));
+        }
```

Comment:
A lot of similar lines might have been useful to implement a function to sort through them